### DISCUSSION TITLE
How to target variants with passthrough?

### DISCUSSION THREAD

**Discussion by justrustyspoons** [primefaces/community#975] [PrimeVue]
Trying to adjust the padding on the small button variant only with global passthrough, is there a way to do this?

**Comment by tugcekucukoglu** [comment]
Yes, you could use global `pt`. For example; https://primevue.org/passthrough/#global

**Reply by justrustyspoons** [reply]
I am using global passthrough, but how to target a specific variant? I specifically want to change the padding ONLY on the ```size="small"``` variant
